### ISSUE TITLE
-f didn't work

### ISSUE THREAD

**Issue by kokizzu** [ariga/atlas#464]
the [docs](https://atlasgo.io/ui/intro) on "Management UI" section said:
```shell
atlas schema apply -d "mysql://root:pass@tcp(localhost:3306)/example" -f atlas.hcl -w
```

but when executed:
```shell
Error: unknown shorthand flag: 'f' in -f
```

version:
```shell
atlas CLI version v0.3.1
https://github.com/ariga/atlas/releases/tag/v0.3.1
```

**Comment by rotemtam** [comment]
Hi @kokizzu ,
Thanks for reaching out.
Copy pasting your snippet on my local Mac works:
```
➜  v0.3.1 atlas schema apply -d "mysql://root:pass@tcp(localhost:3306)/example" -f atlas.hcl -w
Atlas UI available at: http://127.0.0.1:5800/migrations/12884901889
Press Ctrl+C to stop
```

A few more diagnostic questions:

* What OS are you using? 
* Are you able to use `atlas schema apply` without the `-w` flag? 
* Did you download Atlas from the CDN or compile on your own? 
Thanks

**Comment by kokizzu** [comment]
ah right, i was running wrong command parameter XD `atlas schema inspect`
